content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -0,0 +1,85 @@
+---
+linktitle: Override a Prefab
+title: Override a Prefab
+description: Learn how to override a prefab in Open 3D Engine (O3DE).
+weight: 200
+toc: true
+---
+
+We learned in the previous tutorial that when a prefab is open in [Prefab Edit Mode](/docs/learning-guide/tutorials/entities-and-prefabs/entity-and-prefab-basics#edit-a-prefab), changes to its content are automatically propagated to all instances linked to that prefab. Sometimes though, it is useful to alter a prefab instance on its own without affecting the other instances. For example, a level may have multiple instances of the same car prefab, but each car can be a different color.
+
+If just the prefab that's in Prefab Edit mode can expose its content for editing while its nested prefab instances remain closed, how can we isolate changes to a single instance?
```

Comment:
nit: I feel like this is not phrased clearly and can be removed altogether imo.

Reply:
Thanks for reviewing @srikappa-amzn. Do you recommend removing this part?

"If just the prefab that's in Prefab Edit mode can expose its content for editing while its nested prefab instances remain closed, how can we isolate changes to a single instance?"